hyperspy/io_plugins/digital_micrograph.py

```diff
@@ -548,10 +548,15 @@ def names(self):
 
     @property
     def title(self):
-        if "Name" in self.imdict:
-            return self.imdict.Name
-        else:
-            return ''
+        title = self.imdict.get_item("Name")
```

Comment:
What about the following instead:

```python
title = self.imdict.get_item("Name", default="")
title = title if title else ""
```

This takes into account all known possibilities (no Name attr, Name is a str and Name is empty list). If in some file Name contains something else it'll lead to a new bug that we can investigate and fix appropriately.